sdk/core/core-paging/package.json

```diff
@@ -35,15 +35,22 @@
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build": "tsc -p .",
     "build:test": "tsc -p .",
```

Comment:
This can just be "echo skipped" since there is no special command to build test assets